### ISSUE TITLE
 all nodes inherit from the node (pop up) not want to become invisible when I call it a function (hide ())

### ISSUE THREAD

**Issue by OlexiyKravchuk** [godotengine/godot#5014]
the same problem I see in all nodes inherit from the node (pop up),for some reason it does not want to become invisible when I call it a function hide ()

**Comment by volzhs** [comment]
I could reproduce this issue.
Here is what I did.

![image](https://cloud.githubusercontent.com/assets/8281454/15785838/5df2cb06-29f4-11e6-89af-75203fb7b538.png)

Would you post a sample project to reproduce this?

**Comment by akien-mga** [comment]
> I could reproduce this issue.

Could, or couldn't? If you could, I guess your own test could be used as a reproducer?

**Comment by volzhs** [comment]
@akien-mga sorry for mis-typing. **could not reproduce**. I will edit first comment.

**Comment by OlexiyKravchuk** [comment]
I'm sorry guys, it was my mistake, I've created an example in which copied, problematic code from my project, but in the example it worked perfectly, and the long time I could not understand why do not work on the original.
But then I found the reason, it was the function duplicate () located in the function assigned to the paragraph of that same pop-up node, with which I have created a new instance of the node control, next to the original, and that in which have been popup nodes which at this moment have been visible, and then hidden.
But the fact is that it is created and duplicate themselves and the child nodes which visible at this moment, and set them in the same place, where the originals, which have just been successfully hidden.
And the result I was creating the illusion that they can not hide, although it is had already been other nodes in the same place and looking the same.
I realized this when I started to change the duplicates, and with them have begun change, and their lost children popup nodes.
Once again, excuse me for troubling you, I close this issue ...
